src/frontend/handlers.go

```diff
@@ -510,3 +536,36 @@ func stringinSlice(slice []string, val string) bool {
 	}
 	return false
 }
+
+func getDeploymentDetails(httpRequest *http.Request) map[string]string {
```

Comment:
We can move it. But I feel like that's an overkill. A new file in go is a new package. So ideally we would have multiple utility functions put together inside a new package called `utils`. Is that what you suggest?

Reply:
Ahh, okay, if it's overkill, let's not create a separate file.
And I think creating a `utils` module is outside the scope of this pull-request — so that won't be necessary.
Resolved. 👍